project-base/storefront/components/Basic/Table/Table.tsx

```diff
@@ -6,7 +6,7 @@ type TableProps = {
 };
 
 export const Table: FC<TableProps> = ({ head, children, className }) => (
-    <div className={twMergeCustom('overflow-x-auto rounded-xl border-2 border-border p-6', className)}>
+    <div className={twMergeCustom('overflow-x-auto rounded border-2 border-border p-6', className)}>
```

Comment:
I have no problem with this change, as design is not something I focus on. IMHO Shopsys Platform could have zero design on SF, but still, there are stakeholders who care about how it looks. I just want to know if they're on board. Again, you can close this comment as you wish, just make sure this was discussed with Peta, Martin, maybe Vasek, Tom...

Reply:
Yup, don't worry. Everyone will be notified about this new design.